recipe/build.sh

```diff
@@ -34,6 +36,12 @@ if [ "${SHORT_OS_STR}" == "Darwin" ]; then
     CPU_DISPATCH_FLAGS="-DCPU_DISPATCH=SSE4_1;SSE4_2;AVX;FP16"
 fi
 
+if [ "${MACHINE_STR}" == "aarch64" ] || [ "${MACHINE_STR:0:3}" == "arm" ] || [ "${MACHINE_STR:0:3}" == "ppc" ]; then
+    echo Building aarch or ppc
+    OPENMP="-DWITH_OPENMP=1"
```

Comment:
Ok, nitpicking, but is it true that this variable has already been set above on line 15?

Really, you want QT=0 and nothing else right?